### PR TITLE
[15.0][MIG] partner_contact_sale_info_propagation: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36968

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
Waiting for https://github.com/OCA/sale-workflow/pull/2113

File: partner_contact_sale_info_propagation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner contact sale info propagation",
+    "summary": "Propagate Salesperson and Sales Channel " "from Company to Contacts",
+    "version": "15.0.1.0.1",
```
Comment (pedrobaeza) [inline partner_contact_sale_info_propagation/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 could you please review?

**Comment by cesar-tecnativa** [comment]
@ernestotejeda could you please review?

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**